src/System.Net.Http/ref/System.Net.Http.csproj

```diff
@@ -10,6 +10,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Release|AnyCPU'" />
   <ItemGroup>
     <Compile Include="System.Net.Http.cs" />
+    <Compile Include="System.Net.Http.Win32.cs" Condition="'$(TargetGroup)' != 'uap'" />
```

Comment:
This is not `Win32.cs` specific. It is for both Windows and Unix. I think the original name would be fine. Or ifdef in the ref file would be fine too, I think. We have multiple other places with itdefs in ref files.

Reply:
I'll switch this to be `#if uap`